### PR TITLE
refactor Controller remove and load pot

### PR DESCRIPTION
**Pull request by cristianfchq**
-- removePotOverPlateHeater()
-- loadPotOverHeaterPlatePot()
-- sendMessageStatePot()
-- movingPot()

### CODE REVIEW

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Boiler/Boiler.java [hunk 1]
```diff
@@ -31,18 +31,34 @@ public boolean containLiquid() {
      * Changes workingBoiler to true.
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Boiler/Boiler.java]: It seems that this method does more than change workingBolier to true

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Boiler/Boiler.java [hunk 2]
```diff
@@ -31,18 +31,34 @@ public boolean containLiquid() {
...
+        if (waterSensor.getState()) {
+            pressureRelietValve.close();
+            workingBoiler = true;
+            return true;
+        }
+        return false;
     }
 
     /**
      * Changes workingBoiler to false.
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Boiler/Boiler.java]: It seems that this method does more than change workingBolier to false

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Boiler/Boiler.java [hunk 3]
```diff
@@ -31,18 +31,34 @@ public boolean containLiquid() {
...
      */
     @Override
     public void off() {
+        pressureRelietValve.open();
         workingBoiler = false;
     }
 
+    /**
+     *
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Boiler/Boiler.java]: Doc for this method is missing

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java [hunk 4]
```diff
@@ -54,31 +63,56 @@ public void entryOption(final String option) {
         }
     }
 
+    /**
+     * Method for remove or load the pot
+     */
+    public void movingPot(final boolean wantPutPot) {
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java]: The name of this method should be in the infinitive form

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java [hunk 5]
```diff
@@ -54,31 +63,56 @@ public void entryOption(final String option) {
...
+        if (coffeeMaker.isPotInPlace() && !wantPutPot) {
+            actionPot("Removing the pot ...", "The pot has been removed.");
```
Comment (edsonarios) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java]: It is not necessary to create another class just for the message, you could directly use the output class in this place

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java [hunk 6]
```diff
@@ -54,31 +63,56 @@ public void entryOption(final String option) {
...
+            coffeeMaker.removePot();
+        } else {
+            if (!coffeeMaker.isPotInPlace() && wantPutPot) {
+                actionPot("Placing the pot ...", "The pot is ready");
```
Comment (edsonarios) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java]: It is not necessary to create another class just for the message, you could directly use the output class in this place

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java [hunk 7]
```diff
@@ -54,31 +63,56 @@ public void entryOption(final String option) {
...
+            } else {
+                outputs.print(outputs.formatColorGreen(sendMessageStatePot()));
+            }
+        }
+    }
+
+    /**
+     * messages for the action of the pot
+     */
+    public void actionPot(final String action, final String ready) {
```
Comment (edsonarios) [inline src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java]: this method is not encapsulating and it seems redundant to use it

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java (hunk 4 above)
Comment (cristianfchq) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java]: ok

File: src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java (hunk 5 above)
Comment (cristianfchq) [reply src/main/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/Implementation/Controller.java]: ok

**Review by danielaSantaCruz — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]
Good job

**Pull request merged**